esmvalcore/_config/extra_facets/icon-mapping.yml

```diff
@@ -0,0 +1,23 @@
+# Extra facets for native ICON model output
+---
+
+ICON:
+  Amon:
+    clwvi: {raw_name: cllvi}
+  CF3hr:
+    clwvi: {raw_name: cllvi}
+  CFday:
+    clwvi: {raw_name: cllvi}
+  CFsubhr:
+    clwvi: {raw_name: cllvi}
+  E3hr:
+    clwvi: {raw_name: cllvi}
+  fx:
+    areacella:
+      latitude: grid_latitude
+      longitude: grid_longitude
```

Comment:
`grid_latitude` and `grid_longitude` are strictly reserved for rotated grids (see [here](cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html), particularly the note under example 5.6
> A CF compliant application can determine that rlon and rlat are longitude and latitude values in the rotated grid by recognizing the standard names grid_longitude and grid_latitude.

), it might be a good idea to communicate this upstream since that is most likely not what is meant here.

Reply:
Yes, but we are changing the names here. The "problem" here is that `areacella` is read from the "grid information" file which has not been CMORized. All other variables correctly use `latitude` and `longitude`

Reply:
Yeah, I get it. I just thought you could tell the modelers so that they can fix their wrong use of these standard names and we can put a comment that makes us reevaluate the situation in a couple of years.